tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -752,10 +752,7 @@ let SaveAndCheckProject project checker =
 
         let options = project.GetProjectOptions checker
 
-        let! results = checker.ParseAndCheckProject(options)
-
-        if not (Array.isEmpty results.Diagnostics) then
-            failwith $"Project {project.Name} failed initial check: \n%A{results.Diagnostics}"
```

Comment:
Can we put this back? :) Or if not, maybe parametrize if we want to do this check? I think it can cut time on debugging if you know that your initial project already doesn't pass type check.

Reply:
Well yeah, no test will start failing when you remove this check - however, tests that should be failing could start passing at some point :) 

But it's mostly useful when you're writing new ones. 

But it's definitely a valid use case to start with a project that doesn't type check. I'll have a look at how difficult it is to parametrize it.

Reply:
Ok, fixed it for you 😁 One day this will be a full-baked framework!